docs/css/footer.css

```diff
@@ -0,0 +1,63 @@
+#footer {
+
+}
+
+.footer-item {
+	border-bottom: 1px solid var(--color-form);
+	padding: var(--unit-space) var(--unit-gap);
+}
+
+.footer-item:last-child,
+.footer-sub-item:last-child {
+	border: 0 none;
+}
+
+.footer-item-link {
+	text-decoration: none;
+}
+
+.active .footer-item-link,
+.footer-item-link:hover {
+	text-decoration: underline;
+}
+
+#footer .active .footer-item-link {
+	border-bottom-color: var(--color-first);
+	color: var(--color-first);
+	font-weight: 700;
+}
+
+.footer-sub-item {
+	border-bottom: 1px solid var(--color-first);
+	color: var(--color-first);
+	padding: var(--unit-space) var(--unit-gap);
+}
+
+.footer-sub-list {
+	margin-top: var(--unit-gap);
+}
+
+.footer-sub-icons {
+	text-align: right;
+	margin-top: auto;
+}
+
+.footer-icon  {
+	display: inline-block;
+	padding: 5px 20px;
+}
+
+.icon-twitter .icon {
```

Comment:
could also be only icon-twitter and move the class at the html to the svg class="icon icon-twitter".

Reply:
The `.icon` holds definitions that `.icon-twitter` is inheriting, which is the default definition for the icons.